packages/headless/src/features/case-assist/case-assist-slice.test.ts

```diff
@@ -0,0 +1,170 @@
+import {Action} from '@reduxjs/toolkit';
+import {
+  getClassifications,
+  getDocumentSuggestions,
+  setCaseInformationValue,
+} from './case-assist-actions';
+import {caseAssistReducer} from './case-assist-slice';
+import {CaseAssistState, getCaseAssistInitialState} from './case-assist-state';
+
+const rejectWithValue = (action: Action, payload: object) => ({
```

Comment:
This could also fit in a generic util. I've got one in my local branch right now so that's at least one other use case 😄